### DISCUSSION TITLE
Graphical manipulation

### DISCUSSION THREAD

**Discussion by AlineGiroux** [bmcclintock/momentuHMM#117] [Q&A] [answered]
Hi Brett,
I am performing hmm analysis for giant anteaters movement data. I am using hour of the day and environmental temperature as covariates, and I would like to overlap the curves related to different temperatures on the same chart. 

Please see my draft as a example. 
![draft graph](https://user-images.githubusercontent.com/91481371/150591070-6bf133d5-f405-4c4d-b8bd-5d95315ae5d7.jpeg)

However, I don't really know how to extract the objects to manipulate the graph. Could you please help me with that?
Thank you very much

**Comment by bmcclintock** [comment, marked as answer]
See #78 and #98
